File: fe/src/services/api/routes/leaderboard.route.js
import api from "../axios";

export const getLeaderboard = async (params = {}) => {
  const response = await api.get("/leaderboard", { params });
  return response.data;
};

export const getWilayahLeaderboard = async () => {
  const response = await api.get("/leaderboard/wilayah");
  return response.data;
};

File: be/app/api/routes/leaderboard_route.py
"""Leaderboard routes"""
from flask import Blueprint
from app.api.controllers import leaderboard_controller

leaderboard_bp = Blueprint('leaderboard', __name__, url_prefix='/api/leaderboard')

@leaderboard_bp.route('', methods=['GET'])
def get_leaderboard():
    return leaderboard_controller.get_leaderboard()

@leaderboard_bp.route('/wilayah', methods=['GET'])
def get_wilayah_leaderboard():
    return leaderboard_controller.get_wilayah_leaderboard()
